editor/src/core/performance/performance-regression-tests.spec.tsx

```diff
@@ -58,8 +58,8 @@ describe('React Render Count Tests - ', () => {
     )
 
     const renderCountAfter = renderResult.getNumberOfRenders()
-    expect(renderCountAfter - renderCountBefore).toBeGreaterThan(270) // if this breaks, GREAT NEWS but update the test please :)
-    expect(renderCountAfter - renderCountBefore).toBeLessThan(280)
+    expect(renderCountAfter - renderCountBefore).toBeGreaterThan(395) // if this breaks, GREAT NEWS but update the test please :)
+    expect(renderCountAfter - renderCountBefore).toBeLessThan(405)
```

Comment:
YIKES!

Reply:
after testing this for a day, I believe the test is simply more honest now and this is not a true regression